presets/ragengine/config.py

```diff
@@ -19,11 +23,30 @@
 REMOTE_EMBEDDING_URL = os.getenv("REMOTE_EMBEDDING_URL", "http://localhost:5000/embedding")
 REMOTE_EMBEDDING_ACCESS_SECRET = os.getenv("REMOTE_EMBEDDING_ACCESS_SECRET", "default-access-secret")
 
+"""
+=========================================================================
+"""
+
+# Reranking Configuration
+# For now we support simple LLMReranker, future additions would include
+# FlagEmbeddingReranker, SentenceTransformerReranker, CohereReranker
+USE_LLM_RERANK = os.getenv("USE_LLM_RERANK", "false").lower() == "true"
+LLM_RERANKER_BATCH_SIZE = int(os.getenv("LLM_RERANKER_BATCH_SIZE", 5))  # Default LLM batch size
+LLM_RERANKER_TOP_N = int(os.getenv("LLM_RERANKER_TOP_N", 3))  # Default top 3 reranked nodes
+
+"""
+=========================================================================
+"""
+
 # LLM (Large Language Model) configuration
 LLM_INFERENCE_URL = os.getenv("LLM_INFERENCE_URL", "http://localhost:5000/chat")
 LLM_ACCESS_SECRET = os.getenv("LLM_ACCESS_SECRET", "default-access-secret")
 # LLM_RESPONSE_FIELD = os.getenv("LLM_RESPONSE_FIELD", "result")  # Uncomment if needed in the future
 
+"""
+=========================================================================
+"""
+
```

Comment:
low priority: we can reorg these params to cli args and using structured params instead of raw key-value dict.